modules/core/src/main/java/org/apache/ignite/internal/processors/platform/services/PlatformAbstractService.java

```diff
@@ -240,11 +243,13 @@ public void setIgniteInstance(Ignite ignite) {
     @Override public void readExternal(ObjectInput in) throws IOException, ClassNotFoundException {
         svc = in.readObject();
         srvKeepBinary = in.readBoolean();
+        interceptors = in.readObject();
```

Comment:
This will fail if we deserialize an old version of this class (from a node with older Ignite version).

Reply:
AFAIK, Ignite does not provide such type of compatibility (you can't run a cluster with different Ignite versions) - only thin client protocol and persistence provide backwards compatibility.

Reply:
Ok, you are right.